internal/pkg/agent/application/actions/handlers/handler_action_upgrade.go

```diff
@@ -31,12 +34,34 @@ func NewUpgrade(log *logger.Logger, coord *coordinator.Coordinator) *Upgrade {
 }
 
 // Handle handles UPGRADE action.
```

Comment:
Can you add a comment to say the handler does not block/ensures that only 1 upgrade action is running?

Reply:
good idea.  done.